vector/src/debug/java/im/vector/app/features/debug/features/DebugFeaturesSettingsActivity.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.debug.features
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import dagger.hilt.android.AndroidEntryPoint
+import im.vector.app.core.extensions.cleanup
+import im.vector.app.core.extensions.configureWith
+import im.vector.app.databinding.FragmentGenericRecyclerBinding
+import im.vector.app.features.themes.ActivityOtherThemes
+import im.vector.app.features.themes.ThemeUtils
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class DebugFeaturesSettingsActivity : AppCompatActivity() {
```

Comment:
Is there a reason why you are not using VectorBaseActivity? If will provide the ViewBinding for free.

Reply:
I had copied the `sas` package https://github.com/vector-im/element-android/tree/develop/vector/src/debug/java/im/vector/app/features/debug/sas

Reply:
Do not copy paste my bad code please 😆